python/ray/autoscaler/_private/command_runner.py

```diff
@@ -236,16 +234,25 @@ def run_rsync_down(self, source, target, options=None):
                 "rsync failed: '{}'. Falling back to 'kubectl cp'".format(e),
                 UserWarning)
             if target.startswith("~"):
-                target = RAY_HOME + target[1:]
+                target = self._home() + target[1:]
 
             self.process_runner.check_call(self.kubectl + [
                 "cp", "{}/{}:{}".format(self.namespace, self.node_id, source),
                 target
             ])
 
     def remote_shell_command_str(self):
-        return "{} exec -it {} bash".format(" ".join(self.kubectl),
-                                            self.node_id)
+        return "{} exec -it {} -- bash".format(" ".join(self.kubectl),
+                                               self.node_id)
```

Comment:
What is this exactly for?

Reply:
I think this is best practices to separate args from the command

Reply:
Also, currently Kubernetes will give you a deprecation warning if you don'y include the double dash.

Reply:
SGTM!